code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -44,7 +44,9 @@
 	if(user.faction == team)
 		user << "You can't move your own flag!"
 		return
-
+	if(loc == user)
+		if(!user.unEquip(src))
```

Comment:
No it actually unequips from hand A to put it in hand B

If it refuses to leave hand A we return